Runtime/Core/Scripts/Utils/AvatarMeshHelper.cs

```diff
@@ -1,29 +1,63 @@
 using UnityEngine;
+using System.Collections.Generic;
 
 namespace ReadyPlayerMe.Core
 {
     public static class AvatarMeshHelper
     {
-        public static void TransferMesh(GameObject source, SkinnedMeshRenderer[] targetMeshes, Animator targetAnimator)
+        /// <summary>
+        ///     Transfers the mesh and material data from the source to the target avatar.
+        /// </summary>
+        /// <param name="source">Imported Ready Player Me avatar.</param>
+        /// <param name="target">Photon Network Player Character.</param>
+        public static void TransferMesh(GameObject source, GameObject target)
```

Comment:
This is breaking change?

Reply:
Yes changed the signature.

Reply:
What I meant is that this would require a move to the 7.0.0 version, since it might break functionality for the users. Do we want that?

Reply:
This is a minor functionality change probably only used by people who used latest version of photon demo with this method, I would not expect it to be that impactful but you would know better if it is widely used somewhere.

Reply:
I don't on usage, but I rather don't want to find it out 😅

Reply:
Okay now it wont break, I brought old signature, incase anyone still relied on it. Will work and warn that the method is obsolete.